src/customisations/Media.ts

```diff
@@ -82,10 +99,23 @@ export class Media {
      * @param {"scale"|"crop"} mode The desired thumbnailing mode. Defaults to scale.
      * @returns {string} The HTTP URL which points to the thumbnail.
      */
-    public getThumbnailOfSourceHttp(width: number, height: number, mode: 'scale' | 'crop' = "scale"): string {
+    public getThumbnailOfSourceHttp(width: number, height: number, mode: ResizeMode = "scale"): string {
         return MatrixClientPeg.get().mxcUrlToHttp(this.srcMxc, width, height, mode);
     }
 
+    /**
+     * Creates a square thumbnail of the media. If the media has a thumbnail recorded, that MXC will
+     * be used, otherwise the source media will be used.
+     * @param {number} dim The desired width and height.
+     * @returns {string} An HTTP URL for the thumbnail.
+     */
+    public getSquareThumbnailHttp(dim: number): string {
```

Comment:
This sets the `crop` method in many places where previously no method was set. Is it okay?

Reply:
Yes, we were relying on unspecced behavior where the media repo would choose crop for us.